script.js

```diff
@@ -0,0 +1,58 @@
+// dom - selectors
+const rpsBtns = document.querySelectorAll('.rps-btn');
+const chooseContainer = document.querySelector('.choose-container');
+const matchContainer = document.querySelector('.match-container');
+const userSelectionDisplay = document.querySelector('.user-choise');
+// dom - create elements
+const createButton = document.createElement('button');
```

Comment:
opinion:

When I read something that is called `createButton` it sounds a lot like a name for a function creating a button. I would think of a better name for this.

Reply:
I will change it to something more obvious.